recipe/meta.yaml

```diff
@@ -8,34 +7,41 @@ package:
 source:  # [py27 or py3k]
   # Avro exist as two versions in apache avro for py2 and py3.
   fn: avro-{{ version }}.tar.gz  # [py27]
-  url: {{ mirror }}/avro/avro-{{ version }}/py/avro-{{ version }}.tar.gz  # [py27]
-  sha256: 'e4181a9dc686c82cbc9bb496787d659d55582c44ad74b69924c45f80f3d513a1'  # [py27]
+  url: https://files.pythonhosted.org/packages/a1/9a/a2c83f6105d975ec6680b2cab79b4e3bd51ae33f31c10debe8d2485c8f80/avro-{{ version }}.tar.gz  # [py27]
```

Comment:
can you change the url used here to 
```
url: https://pypi.io/packages/source/{{ pip_name[0] }}/{{ pip_name }}/{{ pip_name }}-{{ version }}.tar.gz
```

and add a section near the setters at the top doing

{% set pipname = 'avro' %}  # [py <= 27]
{% set pipname = 'avro-python3' %}  # [py >= 34]

Reply:
Couldn't figure out what the URL structure for `pypi.io/packages` was. Very helpful, thanks!